src/main/java/appeng/core/Registration.java

```diff
@@ -338,9 +327,9 @@ void postInit( final FMLPostInitializationEvent event )
 	{
 		this.registerSpatial( true );
 
-		final IAppEngApi api = AEApi.instance();
+		final Api api = Api.INSTANCE;
```

Comment:
In this case I need access to the Factory, but I am not 100% happy about the flow in which the API is instantiated (static field). Would it be possible to use a different mechanism of exposing the API to mods? Or at the very least only expose it once AE begins it's own initailization? 

What is the reasoning for exporting all blocks/items in the way they are currently exposed in the API. Wouldn't it be sufficient to just expose constants for our registry names?